recipes/micronota/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pip install burrito click
```

Comment:
Is burrito and click in the packaged micronota? I thought it had to be in the requirements section or otherwise it those dependencies wouldn't be available when installing.